README.md

```diff
@@ -96,6 +96,41 @@ build {
 }
 ```
 
+### An example with defined option `resolve_aliases`
+
+When using AWS Systems Manager parameters instead of AMI IDs in launch templates (See the [AWS documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/using-systems-manager-parameters.html)), it is essential to set the `resolve_aliases` option to `true`. This allows the post-processor to resolve the SSM Parameter and retrieve the actual AMI ID associated with it.
+
+If `resolve_aliases` is not set to `true`, the post-processor will not detect the actual AMI ID usage, leading to potential deletion of the AMI and its related snapshot, even if they are still in use by the launch template.
+
+```hcl
+source "amazon-ebs" "example" {
+  region        = "us-east-1"
+  source_ami    = "ami-6869aa05"
+  instance_type = "t2.micro"
+  ssh_username  = "ec2-user"
+  ssh_pty       = true
+  ami_name      = "packer-example ${formatdate("YYYYMMDDhhmmss", timestamp())}"
+  tags = {
+    Amazon_AMI_Management_Identifier = "packer-example"
+  }
+}
+
+build {
+  sources = ["source.amazon-ebs.example"]
+
+  provisioner "shell" {
+    inline = ["echo 'running...'"]
+  }
+
+  post-processor "amazon-ami-management" {
+    regions         = ["us-east-1"]
+    identifier      = "packer-example"
+    keep_releases   = 3
+    resolve_aliases = true
+  }
+}
+```
+
```

Comment:
Thanks for adding a detailed example, but as this is not necessarily the most important behavior for most users I feel a simple explanation of the options would suffice. Any thoughts on removing this example?

Reply:
I feel the detailed explanation given in the example could work as a heads up to the users that didn't knew needed this feature but actually needed it.

Maybe we can remove it, but moving the detailed explanation to the option's configuration section.
![image](https://github.com/user-attachments/assets/8d32bdb3-2784-4fa9-9299-48df2d000daa)